File: app/src/main/java/org/mnmv/sirius/Splash.java
package org.mnmv.sirius;

import android.content.Intent;
import android.os.Bundle;
import androidx.appcompat.app.AppCompatActivity;

public class Splash extends AppCompatActivity {

    //Загрузочный экран
    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);

        Intent intent = new Intent(this, General.class);
        startActivity(intent);
        finish();
    }
}

File: app/src/main/java/org/mnmv/sirius/Splash.kt
package org.mnmv.sirius

import android.content.Intent
import android.os.Bundle
import androidx.appcompat.app.AppCompatActivity

class Splash : AppCompatActivity() {

    //Загрузочный экран
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)

        val intent = Intent(this, General::class.java)
        startActivity(intent)
        finish()
    }
}